setup.py

```diff
@@ -12,7 +12,7 @@
 # - git push
 # - python setup.py sdist bdist_wheel upload --sign
 
-__version__ = '3.4.2'
+__version__ = '3.4.2.1'
```

Comment:
Don't change this, I'll take care of it when I package a release.

Also, when I do change it the new version number will be `3.4.3`, following semver.